testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/client/CIBATest.java

```diff
@@ -124,11 +124,11 @@ public static void setDecoupledAuthenticationRequestUri() {
     private final String TEST_CLIENT_PASSWORD = "password";
 
     private AuthenticationRequestAcknowledgement doBackchannelAuthenticationRequest(String username) throws Exception {
-        return doBackchannelAuthenticationRequest(TEST_CLIENT_NAME, TEST_CLIENT_PASSWORD, username, null);
+        return doBackchannelAuthenticationRequest(TEST_CLIENT_NAME, TEST_CLIENT_PASSWORD, username, null, null);
```

Comment:
It seems that you need to set the newly introduced `backchannel_user_code_parameter` to `true`.
To do so, you need to modify `DescriptionConverter` because this test class communicate with keycloak by Admin REST API or Dynamic Client Registration about the client setting.

Currently, the following error occurred when I ran `CIBATest`.
```
[ERROR] Tests run: 25, Failures: 2, Errors: 0, Skipped: 0, Time elapsed: 42.713 s <<< FAILURE! - in org.keycloak.testsuite.client.CIBATest
[ERROR] testBackchannelAuthenticationFlow(org.keycloak.testsuite.client.CIBATest)  Time elapsed: 0.311 s  <<< FAILURE!
java.lang.AssertionError:

Expected: is "secret code"
     but: was ""
        at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
        at org.junit.Assert.assertThat(Assert.java:956)
        at org.junit.Assert.assertThat(Assert.java:923)
        at org.keycloak.testsuite.client.CIBATest.testBackchannelAuthenticationFlow(CIBATest.java:328)
        at org.keycloak.testsuite.client.CIBATest.testBackchannelAuthenticationFlow(CIBATest.java:856)
```

Reply:
Fixed the test.

Reply:
I've confirmed. Thank you.